server/index.js

```diff
@@ -8,7 +8,8 @@ app.use(express.static(path.join(__dirname, '/../client/dist')));
 
 
 app.get('/items/:itemId/reviews', (req, res) => {
```

Comment:
I didn't realize your component had so many get request

Reply:
Yeah I still need to hook this up the rest of the way. My component is a little weird because I display all the reviews for a given store but we are getting a singular item